app/helpers/claims_helper.rb

```diff
@@ -25,10 +34,10 @@ def eligibility_answers(eligibility)
 
   def verify_answers(claim)
     [].tap do |a|
-      a << ["Full name", claim.full_name]
-      a << ["Address", claim.address] if claim.address_verified?
-      a << ["Date of birth", l(claim.date_of_birth)]
-      a << ["Gender", t("answers.payroll_gender.#{claim.payroll_gender}")] if claim.payroll_gender_verified?
+      a << [I18n.t("verified_fields.full_name").capitalize, claim.full_name]
+      a << [I18n.t("verified_fields.address").capitalize, claim.address("<br>").html_safe] if claim.address_verified?
```

Comment:
`html_safe` issue again.

Reply:
Yeah, running `html_safe` on just the `<br>` results in it returning `&lt;br&gt;` in the string. I think the risks of getting dodgy HTML from Verify service providers, especially considering it's run through the VSP first, it extremely low

Reply:
we can use `simple_format` to convert newlines to breaks.

Reply:
actually, scratch that, I've wrapped it in a `sanitize` call.